src/engine/properties.js

```diff
@@ -226,11 +226,16 @@ function connectSignal(item, signalName, value, objectScope, componentScope) {
     for (const j in item[signalName].parameters) {
       params.push(item[signalName].parameters[j].name);
     }
+    // Wrap value.src in IIFE in case it includes a "return"
     value.src = `(
       function(${params.join(", ")}) {
         QmlWeb.executionContext = __executionContext;
+        var bp = QmlWeb.engine.$basePath;
```

Comment:
Probably another name is needed in case if there is a `bp` property.
E.g. `__basePath` could do.